docker-start.sh

```diff
@@ -15,6 +15,9 @@ else
   chia start farmer
 fi
 
+trap "chia stop all -d; exit 0" SIGINT SIGKILL SIGTERM
+
 # Ensures the log file actually exists, so we can tail successfully
 touch "$CHIA_ROOT/log/debug.log"
-tail -f "$CHIA_ROOT/log/debug.log"
+tail -f "$CHIA_ROOT/log/debug.log" &
+while true; do sleep 30; done
```

Comment:
```suggestion
while true; do sleep 1; done
```

Changing this to 1 in my testing enabled catching the signals from a docker stop, etc

Reply:
Would it be alright for me to try with a higher value as well? Sleeping only 1 second may still cause a decent amount of load on the CPU which should probably be avoided if possible